omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -171,11 +171,17 @@
             # remove windows specific configs
             delete "#{install_dir}/etc/conf.d/winproc.d"
 
+            if ENV['HARDENED_RUNTIME_MAC'] == 'true'
+                hardened_runtime = '-o runtime '
```

Comment:
Note you are using the short form instead of `--options`. It's fine if this works, of course.